From 5e6b89ced7070078eb57272909556a6b0ac124c6 Mon Sep 17 00:00:00 2001
From: Forrest Waters <fwaters@anaconda.com>
Date: Fri, 8 Nov 2019 08:34:49 -0600
Subject: [PATCH] disable homebrew ssl search path

---
 cmake/Modules/FindOpenSSL_EP.cmake | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/cmake/Modules/FindOpenSSL_EP.cmake b/cmake/Modules/FindOpenSSL_EP.cmake
index e371b73c..aa846057 100644
--- a/cmake/Modules/FindOpenSSL_EP.cmake
+++ b/cmake/Modules/FindOpenSSL_EP.cmake
@@ -38,9 +38,9 @@ include(TileDBCommon)
 set(OPENSSL_PATHS ${TILEDB_EP_INSTALL_PREFIX})
 
 # Add /usr/local/opt, as Homebrew sometimes installs it there.
-if (NOT TILEDB_DEPS_NO_DEFAULT_PATH)
-  list(APPEND OPENSSL_PATHS /usr/local/opt/openssl)
-endif()
+#if (NOT TILEDB_DEPS_NO_DEFAULT_PATH)
+#  list(APPEND OPENSSL_PATHS /usr/local/opt/openssl)
+#endif()
 
 # First try the CMake-provided find script.
 set(OPENSSL_ROOT_DIR ${OPENSSL_PATHS})
@@ -133,4 +133,4 @@ endif()
 if (TILEDB_OPENSSL_EP_BUILT AND TILEDB_INSTALL_STATIC_DEPS)
   install_target_libs(OpenSSL::SSL)
   install_target_libs(OpenSSL::Crypto)
-endif()
\ No newline at end of file
+endif()
-- 
2.21.0 (Apple Git-122)

